### DISCUSSION TITLE
pls fix this

### DISCUSSION THREAD

**Discussion by sudobrat** [prasanthrangan/hyprdots#113] [Q&A] [answered]
![image](https://github.com/prasanthrangan/hyprdots/assets/126271100/faa63a61-e0fe-490a-97e4-d14bb7137450)
how can i fix this?

**Comment by prasanthrangan** [comment, marked as answer]
Hi, you'll have to adjust the neofetch config for your res. - [this](https://github.com/prasanthrangan/hyprdots/blob/5e69e3a1485309008cfd430c410d72ab5b354e63/Configs/.config/neofetch/config.conf#L839C1-L839C19) line.
